### DISCUSSION TITLE
I installed Azura without Docker

### DISCUSSION THREAD

**Discussion by heathcliff2019** [AzuraCast/AzuraCast#7492] [General]
First, I want to thank the creator(s) and the community for making this possible. 

Here's the thing. I installed Azuracast on Linode using the [One-click installer](https://www.linode.com/marketplace/apps/linode/azuracast/). Everything is up and running and the next day I have my oops moment. I should have picked the one-click installation with Docker. Can I leave it like it is or do I still need to install docker?

I appreciate your time and help in advance.

**Comment by BusterNeece** [comment]
@heathcliff2019 I was under the impression that all of the Linode one-click VMs were using the Docker installation method. Are they not? Does it list a different version on the site footer?

**Reply by heathcliff2019** [reply]
I'm not sure @BusterNeece

**Reply by LordHelmchen666** [reply]
He mean this footer
![Screenshot_20241005_184413](https://github.com/user-attachments/assets/96b3f85d-e659-427f-ade5-aa6ffefc1cc5)

**Reply by heathcliff2019** [reply]
> He mean this footer ![Screenshot_20241005_184413](https://private-user-images.githubusercontent.com/43477041/373887940-96b3f85d-e659-427f-ade5-aa6ffefc1cc5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjgxNTQwNzAsIm5iZiI6MTcyODE1Mzc3MCwicGF0aCI6Ii80MzQ3NzA0MS8zNzM4ODc5NDAtOTZiM2Y4NWQtZTY1OS00MjdmLWFkZTUtYWE2ZmZlZmMxY2M1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDA1VDE4NDI1MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk5MmQ3YWY1ODgwODA5MzNkMmEzNjNlNzU4MGZlMDA3YjVjNzgwOGVlOTgwMTc2MWZkY2NiNDA1YmJjNTY3NjkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.KEHWnT0hEQjTpx5cz2cB5OgxJrwj78Yew6tHp2fcVDI)

I feel stupid. Thanks @LordHelmchen666 

![suma](https://github.com/user-attachments/assets/9b1e7c14-a67f-4c10-bc19-c40ab48f1dd0)
